xml/common_intro_typografie_i.xml

```diff
@@ -11,20 +11,59 @@
  <title>Documentation Conventions</title>
 
  <info>
-<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-    <dm:maintainer>fs</dm:maintainer>
-    <dm:status>editing</dm:status>
-    <dm:deadline/>
-    <dm:priority/>
-    <dm:translation>no</dm:translation>
-    <dm:languages/>
-</dm:docmanager>
-</info>
-<para>
-  The following typographical conventions are used in this manual:
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>fs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  The following admonition elements and typographical conventions are used
+  in this documentation:
  </para>
 
  <itemizedlist mark="bullet" spacing="normal">
+  <warning>
+   <para>
+    Vital information to be aware of before proceeding. Warns you about
+    security issues, potential loss of data, damage to hardware, or physical
+    hazards.
+   </para>
+  </warning>
+  <important>
+   <para>
+    Important information to be aware of before proceeding.
+   </para>
+  </important>
+  <note>
+   <para>
+    Additional information, for example about differences in software
+    versions.
+   </para>
+  </note>
+  <tip>
+   <para>
+    A guideline or a piece of practical advice.
+   </para>
+  </tip>
+  <listitem>
+<screen>&prompt.user;<command>command</command></screen>
+   <para>
+    Commands than can be run by any user, including the &rootuser; user.
+   </para>
+  </listitem>
+  <listitem>
+<screen>&prompt.root;<command>command</command></screen>
+   <para>
+    Commands that must be run with &rootuser; privileges. Alternatively, you
+    can prefix these commands with the <command>sudo</command> command to
```

Comment:
"...Alternatively, in many cases you can..."
Not sure if that ruins the sentence, though. [I do not have a good example where sudo does not work on SUSE at the moment. But I think I have seen that multiple times.]